onnxruntime/contrib_ops/cpu/transformers/subgraph_gpt.cc

```diff
@@ -27,7 +27,8 @@ Status GptSubgraph::CreateInitialFeeds(
     const GenerationDeviceHelper::AddToFeedsFunc& add_to_feeds_func,
     IAllocatorUniquePtr<char>& buffer,
     Stream* ort_stream,
-    int max_seq_len_past_present_share_buffer) {
+    int max_seq_len_past_present_share_buffer,
```

Comment:
the name looks a bit ambiguous. how about  past_present_share_buffer_max_seq_len, or max_iterations?

Reply:
I have changed it to past_present_share_buffer_max_seq_len. Worth mentioning that this var wasn't introduced in this PR. But I have changed it for now.